testing/acceptance_test.py

```diff
@@ -2,6 +2,7 @@
 import sys
 import textwrap
 import types
+from platform import system
```

Comment:
Hmm first time I see this module!

Usually `sys.platform.startswith("win")` is used for checking Windows, is there a difference?

Reply:
From https://stackoverflow.com/a/1857/7262247 (and https://docs.python.org/3/library/platform.html#platform.system)

Reply:
Thanks for the reference!

I think we should still use `sys` though, because this is what the rest of the code uses, and doesn't require importing a new module (sys is already imported), and mypy has special logic which understand `sys.platform` checks.

Reply:
ok done.